extensions/resteasy-jsonb/deployment/src/main/java/io/quarkus/resteasy/jsonb/deployment/ResteasyJsonbProcessor.java

```diff
@@ -15,23 +15,14 @@
  */
 package io.quarkus.resteasy.jsonb.deployment;
 
-import org.eclipse.yasson.JsonBindingProvider;
-import org.glassfish.json.JsonProviderImpl;
-
 import io.quarkus.deployment.annotations.BuildProducer;
 import io.quarkus.deployment.annotations.BuildStep;
 import io.quarkus.deployment.builditem.FeatureBuildItem;
-import io.quarkus.deployment.builditem.substrate.ReflectiveClassBuildItem;
 
 public class ResteasyJsonbProcessor {
```

Comment:
Do we have to keep this one?

Reply:
Yes, I think it's better to have it as it simplifies using RESTEasy + JSON which is a **very** common use case.